products/compute/api.yaml

```diff
@@ -779,8 +779,6 @@ objects:
                 either maxRate or maxRatePerEndpoint must be set.
             - !ruby/object:Api::Type::Double
               name: 'maxUtilization'
-              send_empty_value: true
```

Comment:
Removing this means it can no longer be updated to 0 (or set explicitly to 0, if there's a server-side default that's nonzero). is that what we want?